scraper/sprinters_driver.py

```diff
@@ -0,0 +1,88 @@
+from selenium import webdriver
+from selenium.webdriver.chrome.service import Service as ChromeService
+from webdriver_manager.chrome import ChromeDriverManager
+from selenium.webdriver.edge.service import Service as EdgeService
+from webdriver_manager.microsoft import EdgeChromiumDriverManager
+from webdriver_manager.core.utils import ChromeType
+from selenium.webdriver.chrome.service import Service as BraveService
+from selenium.webdriver.common.by import By
+from selenium.webdriver.remote.webelement import WebElement
+from scraper_utils import get_random_number_from_range
+from selenium.common.exceptions import NoSuchElementException, ElementClickInterceptedException
+from typing import List
+
+class SprintersDriver:
+    def __init__(self) -> None:
+        browser = 1#get_random_number_from_range(1, 3)
```

Comment:
should browser always be 1?

Reply:
tested it on the chrome browser, downloads, launching worked as well on my machine, but to speed things up deside to choose 1, i know for now it works fine on browser one

Reply:
tried also option 2 and 3 in limited environment and browser 3 works fine as well (probably because if also works on Chrome webdriver)

option 2 raised while trying to close cookies using second path -> preferences -> save preferences

`Message: element click intercepted: Element is not clickable at point (285, 963)`

first path works as well

Reply:
done in https://github.com/230443/Sprinters/pull/15/commits/9e42bd68aef8258c4998e4a938065f396967f483